README.md

```diff
@@ -9,10 +9,14 @@ https://wweitzel.github.io/ngx-joystick/
 ---
 
 ## Installation
-For Angular 15 projects
+For Angular 16 projects
 ```
 npm install ngx-joystick
 ```
+For Angular 15 projects
+```
+npm install ngx-joystick@3.0.1
```

Comment:
This should be 3.0.2 since that is current version on npm right now https://www.npmjs.com/package/ngx-joystick